flow/statefulflow.yaml

```diff
@@ -0,0 +1,23 @@
+components:
+  schemas:
+    StatefulFlow:
+      description: |-
+        A high level data plane traffic flow.
+      type: object
+      #required: [tx_rx, name]
+      properties:
+        choice:
+          description: |-
+            Stateful traffic configuration.
+          type: string
+          x-enum:
+            rocev2:
+              x-field-uid: 1
+          x-field-uid: 1
+        rocev2:
+          description: |-
+            RoCEv2 Flow Groups.
```

Comment:
@kunkeysight is this description sufficient for a first time reader to make out what this attribute means ? Suggestion is to make it little more descriptive.